### ISSUE TITLE
Missing some letters in a long formula

### ISSUE THREAD

**Issue by scrouthtv** [xournalpp/xournalpp#3407]
**Affects versions :**
 - OS: Windows 10 64-bit (can't reproduce on Arch Linux)
 - Which version of libgtk do you use 3.24.29
 - Version of Xournal++: 1.1.0 / 9d1277de
 - Installation method: official release

**Describe the bug**
A long formula is consistently rendered with some letters missing.

**To Reproduce**
1. Create a new document
2. Add a LaTeX formula
3. Enter `\Delta h_2 = \frac{S_2 h_2}{E_SA_S} +\alpha_\Theta \Delta\Theta_2 h_2 \qquad (\text{III})` as the formula
4. Add the formula

**Expected behavior**
All letters would appear in the formula.

**Screenshots of Problem**
![image](https://user-images.githubusercontent.com/23227405/133923182-0b801c3a-fa13-4736-8f5a-33a6b083f344.png)

**Additional context**
I can consistently create this error on my Windows machine, it however works fine when using Linux instead.

**Comment by scrouthtv** [comment]
Also happens for `\Delta h = \varepsilon_{33} h = \frac{1+\nu}{1-\nu} \alpha_\Theta \Delta\Theta h`

**Comment by LittleHuba** [comment]
Seems to me as if all characters followed by a `_` are missing? Or are there more characters missing?

**Comment by scrouthtv** [comment]
In the first example, the theta is rendered correctly. I'd rather say that all latin characters followed by a `_` are missing

**Comment by Technius** [comment]
Do you also observe this problem with other LaTeX programs on Windows?

**Comment by scrouthtv** [comment]
These two formulas work fine when put into a simple document and compiled using `pdflatex`. I don't use any other latex programs

**Comment by LittleHuba** [comment]
We compile any latex using `pdflatex` with this template:
https://github.com/xournalpp/xournalpp/blob/master/resources/default_template.tex

You'll have to replace `%%XPP_TEXT_COLOR%%` on line 17 with the color of your choice and `%%XPP_TOOL_INPUT%%` on line 23 with the formula. Could you try to compile your example formulas manually using this template?

**Comment by LittleHuba** [comment]
Okay then it is something on our side. Thanks for the confirmation.

**Comment by scrouthtv** [comment]
One more: `w'''(0) \overset!= 0`

**Comment by LittleHuba** [comment]
Should that version also experience this bug, please also try the nightly version. I currently assume it is some kind of dependency issue that might be fixed in a more recent nightly release.

**Comment by LittleHuba** [comment]
Could you please try the installer of #3477? It might have an impact on this issue as well.

**Comment by scrouthtv** [comment]
Still not working with reinstalled miktex.

Startup:
```

(process:6736): GLib-GIO-WARNING **: 13:13:05.707: Unexpectedly, UWP app `LEGOEducation.EV3ClassroomLEGOEducation_1.5.0.0_x64__by3p0hsm2jzfy' (AUMId `LEGOEducation.EV3ClassroomLEGOEducation_by3p0hsm2jzfy!EV3.EDU') supports 1 extensions but has no verbs

(xournalpp.exe:6736): GLib-GIO-WARNING **: 13:13:07.115: win32 session dbus binary not found: D:\Program Files\Xournal++\bin\\gdbus.exe
** Message: 13:13:07.859: TEXTDOMAINDIR = (NULL), PACKAGE_LOCALE_DIR = ../share/locale/, chosen directory = ../share/locale/

** (xournalpp.exe:6736): WARNING **: 13:13:07.900: XournalMain: System default locale could not be set.
 - Caused by: locale::facet::_S_create_c_locale name not valid
 - Note that it is not supported to set the locale using mingw-w64 on windows.
 - This could be solved by compiling xournalpp with msvc
** Message: 13:13:07.981: Plugin "MigrateFontSizes" UI initialized
```

There's no more output while I enter the formula.

**Comment by NelkonParker** [comment]
I'm guessing that the behaviour in the video attached is an example of this problem? 

This is using hotfix 1.1.0 on Windows 10 with MikTex 2.9

Has there been any progress with this It's a bit of a showstopper for me, as I use latex very frequently.

https://user-images.githubusercontent.com/88211140/144415837-bb264480-2e7f-485c-b1b8-02051be17902.mp4

**Comment by DUOLabs333** [comment]
Does using TexLive help?

**Comment by NelkonParker** [comment]
> Does using TexLive help?

I've never heard of TexLive, I'm afraid, so I don't know. I will look at it when I get some time.

I seem to have found a workaround for the specific problem that I ran into earlier; I have no idea if it is a more general fix though, as I can't reproduce this problem at will. I've attached a video.

https://user-images.githubusercontent.com/88211140/144458219-26791020-727a-4d8a-b215-99cabd41c555.mp4

**Comment by scrouthtv** [comment]
I can check TL as I get home (in an hour or two)

**Comment by DUOLabs333** [comment]
> > Does using TexLive help?
> 
> I've never heard of TexLive, I'm afraid, so I don't know. I will look at it when I get some time.
> 
> I seem to have found a workaround for the specific problem that I ran into earlier; I have no idea if it is a more general fix though, as I can't reproduce this problem at will. I've attached a video.
> 2021-12-02.16-00-20.mp4

Yes, when I was on Windows, I did something similar (making new lines or making no-space characters).
